servicetalk-opentracing-log4j2/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
   implementation project(":servicetalk-opentracing-asynccontext")
   implementation "com.google.code.findbugs:jsr305:$jsr305Version"
   implementation "org.apache.logging.log4j:log4j-api:$log4jVersion"
+  implementation "org.apache.logging.log4j:log4j-core:$log4jVersion"
```

Comment:
If you added `log4j-core` here, you can remove `log4j-api` from the line above

Reply:
Using bom to define a version is one part, we should also limit what dependencies are defined explicitly. If the `log4j-core` is here, `log4j-api` is not required in line 30

Reply:
Resolved in #2217